MIGRATION.md

```diff
@@ -70,37 +75,93 @@
 
 For NPM
 
-```sh
-npm install @contentul/f36-components@beta
-npm install @contentful/f36-icons@beta
+```bash
+npm install @contentul/f36-components
+npm install @contentful/f36-tokens
 ```
 
 For YARN
 
-```sh
-yarn add @contentul/f36-components@beta
-yarn add @contentful/f36-icons@beta
+```bash
+yarn add @contentul/f36-components
+yarn add @contentful/f36-tokens
+```
+
+#### Install icons package separately
+
+Starting from version 4, icons are not included in the main components package. If you need to use our icons or you use them already in version 3, install them separately, by running the following commands:
+
+```bash
+npm install @contentul/f36-icons
 ```
 
-You can install separate packages and use chosen components separately.
-For example, if you need only the `Button` component in your project, you can add this package by running the following command:
+For YARN
+
+```bash
+yarn add @contentul/f36-icons
+```
+
+#### Install packages separately
+
+If for some reason, your project requires installing packages separately it is possible in version 4.
+For example, if you need only the `Button` component in your project, you can add only this package by running the following command:
+
+For NPM
+
+```bash
+npm install @contentul/f36-button
+```
+
+For YARN
+
+```bash
+yarn add @contentul/f36-button
+```
+
+Note: We don't recommend using it this way, it is much easier to [install full package of components](#Step-1-install-new-packages). Tree-shaking will take care of your build
+
+### Step 2 - migrate your components
+
+We created codemods for most of the components to make it easier for you to migrate. All you need to do is run this command and follow the prompt:
+
+`npx @contentful/f36-codemod`
+
+NOTE: If you run this command for the first time it will install the package in the npm cache so you would need to go ahead and run it again.
+
+There are still a couple of components that requires manual migration, have a look on [the detailed documentation how to do it](#changes-per-component-in-v4).
+
+### Step 3 - remove version 3 packages and CSS imports
+
+Now that you have your fresh version of Forma 36 installed, you don't need the old packages anymore. Go ahead and remove them by running the following command:
 
 For NPM
 
-```sh
-npm install @contentul/f36-button@beta
+```bash
+npm uninstall @contentul/f36-components
+npm uninstall @contentful/f36-tokens
+npm uninstall @contentful/f36-fcss
 ```
 
 For YARN
 
-```sh
-yarn add @contentul/f36-button@beta
+```bash
+yarn remove @contentul/f36-components
+yarn remove @contentful/f36-tokens
+yarn remove @contentful/f36-fcss
+```
+
+Now find the following imports and remove them as well:
+
+```tsx
+import '@contentful/forma-36-react-components/dist/styles.css';
+import '@contentful/forma-36-fcss/dist/styles.css';
+import '@contentful/forma-36-tokens/dist/css/index.css';
```

Comment:
Should we also add that user needs to add  our new`GlobalStyles` component that has normalize and some basic styles?

Reply:
AA! Good point!